<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Among Us Toolkit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="header">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Among Us Toolkit</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="pagesWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>4</number>
       </property>
       <widget class="QWidget" name="join_tab">
        <attribute name="title">
         <string>Join a lobby</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>581</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="join_grid">
          <item row="1" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="joinLineEdit">
            <property name="text">
             <string>Code</string>
            </property>
            <property name="maxLength">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="joinButton">
            <property name="text">
             <string>Join</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoRepeat">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <property name="autoRepeatDelay">
             <number>40000</number>
            </property>
            <property name="autoRepeatInterval">
             <number>4000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="sentences_tab">
        <attribute name="title">
         <string>Sentences</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>581</width>
           <height>191</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="sentences_rows">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Add New Pack</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Rename</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>Remove This Pack</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="send_button">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QComboBox" name="sentence_selector_combobox"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="text_to_send_textEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tasks_tab">
        <attribute name="title">
         <string>Tasks</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>591</width>
           <height>201</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <layout class="QGridLayout" name="tasks_grid">
          <item row="1" column="2">
           <widget class="QPushButton" name="divert1_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Divert Power</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="course_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Chart Course</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="align_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Align Engine</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="vent_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Clean Vent</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="wires_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Fix Wiring</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="filter_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Clean Filter</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="shields_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Prime Shields</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="distributor_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Calibrate Distributor</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="asteroids_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Clear Asteroids</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="detect_button">
              <property name="text">
               <string>Detect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="killPushButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>COVID</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
              <property name="autoRepeatDelay">
               <number>100</number>
              </property>
              <property name="autoRepeatInterval">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="trash_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Empty Chute</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="reactor_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Start Reactor</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="manifolds_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Unlock Manifolds</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="scan_button">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Submit Scan</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="stabilize_button">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Stabilize Steering</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="sample_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Inspect Sample</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="divert2_button">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Divert Accept</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="fuel_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Fuel Engines</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="swipe_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Swipe Card</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="download_button">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Download/Upload Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="settings_page">
        <attribute name="title">
         <string>Settings</string>
        </attribute>
        <widget class="QTabWidget" name="tabWidget">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>9</y>
           <width>581</width>
           <height>191</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="coordinates_tab">
          <attribute name="title">
           <string>Coordinates</string>
          </attribute>
          <widget class="QWidget" name="gridLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>562</width>
             <height>141</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="2">
             <widget class="QSpinBox" name="crossYSpinBox">
              <property name="maximum">
               <number>4095</number>
              </property>
             </widget>
            </item>
            <item row="3" column="7">
             <widget class="QPushButton" name="pushButton_6">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Get</string>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QSpinBox" name="displayYSpinBox">
              <property name="maximum">
               <number>4095</number>
              </property>
             </widget>
            </item>
            <item row="2" column="7">
             <widget class="QPushButton" name="pushButton_5">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Get</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="pushButton_2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Get</string>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QSpinBox" name="arrowXSpinBox">
              <property name="maximum">
               <number>4095</number>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QSpinBox" name="tb1YSpinBox">
              <property name="maximum">
               <number>4095</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QSpinBox" name="originYSpinBox">
              <property name="maximum">
               <number>4095</number>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QPushButton" name="pushButton_3">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Get</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="originCoordsButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Get</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="tb1Label">
              <property name="text">
               <string>Textbox 1</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="displayLabel">
              <property name="text">
               <string>Display</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QSpinBox" name="displayXSpinBox">
              <property name="maximum">
               <number>4095</number>
              </property>
             </widget>
            </item>
            <item row="0" column="7">
             <widget class="QPushButton" name="getDisplayButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Get</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="originXSpinBox">
              <property name="maximum">
               <number>4095</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="originLabel">
              <property name="text">
               <string>Origin</string>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QSpinBox" name="tb2XSpinBox">
              <property name="maximum">
               <number>4095</number>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="crossXSpinBox">
              <property name="maximum">
               <number>4095</number>
              </property>
             </widget>
            </item>
            <item row="3" column="6">
             <widget class="QSpinBox" name="tb2YSpinBox">
              <property name="maximum">
               <number>4095</number>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="tb1XSpinBox">
              <property name="maximum">
               <number>4095</number>
              </property>
             </widget>
            </item>
            <item row="2" column="6">
             <widget class="QSpinBox" name="arrowYSpinBox">
              <property name="maximum">
               <number>4095</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="crossLabel">
              <property name="text">
               <string>Cross</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QLabel" name="arrowLabel">
              <property name="text">
               <string>Arrow</string>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QLabel" name="tb2Label">
              <property name="text">
               <string>Textbox 2</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="cooldowns_tab">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <attribute name="title">
           <string>Cooldowns</string>
          </attribute>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="about_tab">
        <attribute name="title">
         <string>About</string>
        </attribute>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>581</width>
           <height>191</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="rowWrapPolicy">
           <enum>QFormLayout::WrapLongRows</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Author:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Rait Kulbok</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Version:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>20220203ALPHA</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
